springwolf-core/src/main/java/io/github/stavshamir/springwolf/configuration/DefaultAsyncApiDocketService.java

```diff
@@ -25,21 +25,17 @@ public class DefaultAsyncApiDocketService implements AsyncApiDocketService {
     /**
      * Docket definition in application.properties
      */
-    private final Optional<SpringwolfConfigProperties> configProperties;
```

Comment:
Optional &lt;SpringwolfConfigProperties&gt;  not necessary, `SpringwolfConfigProperties` are always present (explicitly defined as @Bean)